env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
     log.setDefaultLevel('debug');
   } else {
     log.setDefaultLevel('info');
   }
 
   return {
+    legacyMode: process.env.LEGACY_MODE || false,
```

Comment:
Yeah Bryan and I talked about this at length. I wasn't really comfortable with version detection if there was a viable feature detection alternative. I can definitely see a 3.0 feature that adds views that aren't relevant to 2.6 so anything that's future proof will do.